### ISSUE TITLE
Saving configuration in routers nodes topology and hidden buttons. Is there a way?

### ISSUE THREAD

**Issue by maaz06** [coreemu/core#680]
Hello,
I am using CORE 8.0.0 running on ubuntu 20.04 LTS Virtual machine. And I am using FRR service to construct network topologies. I would like to report that I cannot save the node configurations in network topologies.

For example: If I create a network router node, say "n1.conf", and write configurations for it, I would like those to be saved somewhere say along with the saved network topology file "top.imn". But there is no option in legacy Core GUI (8..0.0) at least to accomplish it
What happens now, whenever, I reload my topology "top.imn", the configurations in "n1.conf" are automatically generated. And I have to replace them during runtime. 

**My question is: Is there a way I can save manually written configurations for each router node in my CORE network topology somehow.**

Below is the image showing textbox where configuration although can be written but **cannot be saved/applied** to topology. They revert back to automatically generated configurations each time i restart the 'Node n1 (n1) services' dialogue box and click on FRRzebra icon.

![image](https://user-images.githubusercontent.com/56983195/161040591-25cbe089-0d05-419e-8582-73fe18538735.png)

**Comment by maaz06** [comment]
Yes. I am missing all the buttons.
What do you suggest i should change to solve this issue. I upgraded from Core 7.5.2 to 8.0. Before, i also did not have any buttons.

Any suggestions to solve the problem will be helpful. Thanks !!

**Comment by bharnden** [comment]
I can see you are using VirtualBox and I recall there being an issue related to the settings being used there for graphics. I forget offhand and could not find it quickly, but it may be worth trying different graphics settings for the VM as a start.

**Comment by maaz06** [comment]
Hi again, 
Yes, I am using Virtualbox and It does seem the problem is with the graphics and display settings. I did not change the graphics settings in Virtualbox, but I changed my display.
**And Now, now I can see the missing buttons when I changed my display settings to higher resolution from 1366x768 to 1680x1050.** (I added an extended monitor (1680x1050) with 12" laptop (1366x768 display) and I can see buttons there, but not on my laptop screen. 
Thanks for the tip. :)
